datalad/core/local/tests/test_diff.py

```diff
@@ -61,8 +64,10 @@ def test_magic_number():
     # commit
     # given the level of dark magic, we better test whether this stays
     # constant across Git versions (it should!)
-    out, err = GitRunner().run('cd ./ | git hash-object --stdin -t tree')
-    eq_(out.strip(), PRE_INIT_COMMIT_SHA)
+    out = GitWitlessRunner().run(
+        ['git', 'hash-object', '-t', 'tree', '/dev/null'],
```

Comment:
*nix only? or if there is a pipe `|` there is a guarantee to have `/dev/null`?

Reply:
Yes, this test did not and does not run on windows.

Reply:
Good catch. I copied this too blindly from the tests of the old `diff`, where the test isn't cross-platform. Fixed.